interface/usergroup/usergroup_admin_add.php

```diff
@@ -645,6 +655,51 @@ function authorized_clicked() {
 
 });
 </script>
+
+<!-- Password Strength Meter JavaScript -->
+<script>
+function checkPasswordStrength() {
+    var number = /(\p{N})/u;
+    var alphabets = /(\p{L})/u;
+    var special_characters = /([~,!,@,#,$,%,^,&,*,-,_,+,=,?,>,<])/;
+
+    var pwd = document.getElementById("stiltskin").value;
+    var strength = 0;
+
+    if (pwd.length < 6) {
+        document.getElementById('password_strength_meter').style.backgroundColor = "#ff6666";
+        document.getElementById('password_strength_text').innerHTML = "<?php echo xlt('Very Weak'); ?>";
```

Comment:
innerText

also use xlj instead (and note dropping of the outside quotes (also do this for the 3 lines below):
```
document.getElementById('password_strength_text').innerHTML = <?php echo xlj('Very Weak'); ?>;
```

Reply:
Fixed the issue